tests/func/test_import_url.py

```diff
@@ -398,3 +398,18 @@ def test_import_url_empty_directory(tmp_dir, dvc, workspace):
     empty_dir = tmp_dir / "empty_dir"
     assert empty_dir.is_dir()
     assert tuple(empty_dir.iterdir()) == ()
+
+
+def test_import_url_to_remote_status(tmp_dir, dvc, local_cloud, local_remote):
+    local_cloud.gen("foo", "foo")
+
+    stage = dvc.imp_url(str(local_cloud / "foo"), to_remote=True)
```

Comment:
It can be used in this way `str(Path)`? In the previous I'm always using `os.fspath(Path)` following other's work.

Reply:
@karajan1001 either one can be used (see https://github.com/iterative/dvc/blob/master/dvc/path_info.py#L50)